File: android/src/main/java/com/bigbang/wallet/RNWalletOptions.java
package com.bigbang.wallet;

public class RNWalletOptions {
	private boolean beta;
	private boolean shareAccountWithParentChain;
	private boolean BBCUseStandardBip44ID;
	private boolean MKFUseBBCBip44ID;

	public boolean isBeta() {
		return beta;
	}

	public void setBeta(boolean beta) {
		this.beta = beta;
	}

	public boolean isShareAccountWithParentChain() {
		return shareAccountWithParentChain;
	}

	public void setShareAccountWithParentChain(boolean shareAccountWithParentChain) {
		this.shareAccountWithParentChain = shareAccountWithParentChain;
	}

	public boolean isBBCUseStandardBip44ID() {
		return BBCUseStandardBip44ID;
	}

	public void setBBCUseStandardBip44ID(boolean BBCUseStandardBip44ID) {
		this.BBCUseStandardBip44ID = BBCUseStandardBip44ID;
	}

	public boolean isMKFUseBBCBip44ID() {
		return MKFUseBBCBip44ID;
	}

	public void setMKFUseBBCBip44ID(boolean MKFUseBBCBip44ID) {
		this.MKFUseBBCBip44ID = MKFUseBBCBip44ID;
	}
}

File: ios/RNWalletOptions.m
//
//  RNWalletOptions.m
//  appcenter-analytics
//
//  Created by wuye on 2020/9/22.
//
#import <RNWalletOptions.h>
@implementation RNWalletOptions
- (void)setBeta:(BOOL)beta{
    _beta = beta;
}
- (BOOL)beta{
    return _beta;
}

- (void)setShareAccountWithParentChain:(BOOL)shareAccountWithParentChain{
    _shareAccountWithParentChain = shareAccountWithParentChain;
}
- (BOOL)shareAccountWithParentChain{
    return _shareAccountWithParentChain;
}

- (void)setMKFUseBBCBip44ID:(BOOL)MKFUseBBCBip44ID{
    _MKFUseBBCBip44ID = MKFUseBBCBip44ID;
}
- (BOOL)MKFUseBBCBip44ID{
    return _MKFUseBBCBip44ID;
}

- (void)setBBCUseStandardBip44ID:(BOOL)BBCUseStandardBip44ID{
    _BBCUseStandardBip44ID = BBCUseStandardBip44ID;
}
- (BOOL)BBCUseStandardBip44ID{
    return _BBCUseStandardBip44ID;
}
@end
